tox.ini

```diff
@@ -15,6 +15,9 @@ commands=nosetests
 [testenv:py35]
 commands=nosetests
 
+[testenv:py36]
+commands=nosetests
```

Comment:
FYI - `nose` is no longer maintained. It's been replaced by `nose2`; alternatively `pytest` is also an awesome choice and easy to migrate too.

For reference, here's a PR against one of my projects (https://github.com/TestInABox/stackInABox/pull/74) for converting it from `nosetests` to `pytest`, the only changes are in the test requirements and tox.ini and are pretty minimal.

$0.02